splunklogger.js

```diff
@@ -50,6 +50,7 @@ function _err(err, context) {
  * @property {object} config - Configuration settings for this <code>SplunkLogger</code> instance.
  * @property {function[]} middlewares - Middleware functions to run before sending data to Splunk.
  * @property {object[]} contextQueue - Queue of <code>context</code> objects to be sent to Splunk.
+ * @property {number[]} eventSizes - Parallel queue of the sizes of events stored in <code>contextQueue</code>.
```

Comment:
I'm not sure what is parallel queue... I saw parallel queues in Costco. Is that what you mean? :)